pup-0004.md

```diff
@@ -93,10 +93,10 @@ available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.ht
 
 ## Displaying the CoC
 
-A link to the CoC above will be provided in the "Code of Conduct" section of
-CONTRIBUTING.md file in the root of the pulp Git repository. Each Pulp plugin
-will be encouraged to include a link to the CoC above in a CONTRIBUTING.md file
-in the root of its repository.
+The text of the CoC will reside on our website at https://pulpproject.org/conduct/
```

Comment:
Missing a `.`.

Reply:
The period was deliberately omitted so as not to get in the way of the link.  If you think "/conduct/." is not confusing, it can be added.

Reply:
I see. What about:

```
The text of the CoC will reside on [our website](https://pulpproject.org/conduct/).
```

Reply:
I see. What about maybe quoting it (ie `reside on our website at "https://pulpproject.org/conduct/".`)?

Reply:
I'm fine with leaving it as is too.